_benchmark/quickstart.md

```diff
@@ -14,21 +14,52 @@ To perform the Quickstart steps, you'll need to fulfill the following prerequisi
 
 - A currently active OpenSearch cluster. For instructions on how to create an OpenSearch cluster, see [Creating a cluster]({{site.url}}{{site.baseurl}}//tuning-your-cluster/index/).
 - Git 2.3 or greater.
+- Python 3.8 or later
 
-Additional prerequisites are required, depending on your installation method:
+## Set up an OpenSearch cluster
 
-- If you plan to install OpenSearch Benchmark with [PyPi](https://pypi.org/), install Python 3.8 or later.
-- If you plan to install OpenSearch Benchmark using Docker, install Docker. 
+If you don't already have an acitive OpenSearch cluster, you can launch a new cluster to being using OpenSearch Benchmark.
 
-## Installing OpenSearch Benchmark
+- Using **Docker Compose**. For instructions on how to use Docker Compose, see [OpenSearch Quickstart]({{site.url}}{{site.baseurl}}/quickstart/).
+- Using **Tar**. For instructions on how to install OpenSearch with Tar, see [Installing OpenSearch > Tarball]({{site.url}}{{site.baseurl}}/install-opensearch/tar/#step-1-download-and-unpack-opensearch/).
+
+OpenSearch Benchmark has not been tested with the Window's distribution of OpenSearch.
+{: .note}
+
+After instllation, you can verify OpenSearch is running by going to `localhost:9200`. If you're running your cluster with the security plugin enabled, OpenSearch will expect SSL connections with the username "admin" and password "admin".  However, since the localhost address is not a unique public address, no certificate authority will issue an SSL certificate for it, so certificate checking will need to be disabled using the `-k` option.
```

Comment:
```suggestion
After instillation, you can verify OpenSearch is running by going to `localhost:9200`. If you're running your cluster with the security plugin enabled, OpenSearch will expect SSL connections with the username "admin" and password "admin".  However, since the localhost address is not a unique public address, no certificate authority will issue an SSL certificate for it, so certificate checking will need to be disabled using the `-k` option.
```